docs/fe-style-guide.md

```diff
@@ -0,0 +1,71 @@
+# Bytebase Frontend Style Guide
+
+We are following [Bytebase's API Style Guide](https://github.com/bytebase/bytebase/blob/main/docs/api-style-guide.md) to ensure our style guide is **consistent**.
+
+# URL naming
+
+## Use slugs in URLs
+
+Bytebase uses [slugs](https://en.wikipedia.org/wiki/Clean_URL#Slug) in URLs for resource identification. We use `/issue/hello-world-101` instead of `/issue/101`
+
+_Rationale_: This make URLs more clean and human-readable, especially when sharing via social medias.
+
+## Use lower case, kebab-case for phrases
+
+Use `/anomaly-center` instead of `/anomalycenter` or `/anomalyCenter`
+
+_Rationale_: Using `/anomalycenter` makes it more difficult to read. `/anomalyCenter` makes it more difficult to memoize and input because of mixed cases. Using `/anomaly-center` is good for readability and improves consistency with slugs.
```

Comment:
```suggestion
_Rationale_: Using `/anomalycenter` makes it more difficult to read. `/anomalyCenter` makes it more difficult to memoize and type because of mixed cases. Using `/anomaly-center` is good for readability and improves consistency with slugs.
```